src/main/resources/icons/files/gitlab.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+    <polygon points="16 28.896 16 28.896 21.156 13.029 10.844 13.029 16 28.896" style="fill:#e24329"/>
```

Comment:
Please set the icons on a single color (select whichever you want)

Reply:
You mean for all `polygon` and `path` tags?

Reply:
what do you mean?

Reply:
Color was updated. 
I mean that I don't know what will be with `git` and `gitlab` icon patterns now because `git` trying to take all files starting with `.git` and can override `gitlab`.

Reply:
So Gitlab should appear before Git

Reply:
Done